RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -42,30 +42,40 @@ public struct CustomerCenterView: View {
     private var colorScheme
 
     private let mode: CustomerCenterPresentationMode
+    private let isInNavigationStack: Bool
 
     /// Create a view to handle common customer support tasks
     /// - Parameters:
     ///   - customerCenterActionHandler: An optional `CustomerCenterActionHandler` to handle actions
     ///   from the Customer Center.
-    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil) {
-        self.init(customerCenterActionHandler: customerCenterActionHandler, mode: .default)
+    ///   - isInNavigationStack: Whether this view is already inside a navigation stack
+    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+                isInNavigationStack: Bool = false) {
```

Comment:
Yes, I think if it's not possible to detect automatically, this boolean makes sense. The only thing I see is that `NavigationStack` is an iOS 16 concept, and before it was `NavigationView`